src/main/kotlin/com/workos/mfa/MfaApi.kt

```diff
@@ -247,6 +250,76 @@ class MfaApi(private val workos: WorkOS) {
     return workos.post("/auth/factors/verify", VerifyFactorResponse::class.java, config)
   }
 
+  /**
+   * Parameters for the [verifyChallenge] method.
+   */
+  @JsonInclude(Include.NON_NULL)
+  class VerifyChallengeOptions constructor(
+    @JsonProperty("authentication_challenge_id")
```

Comment:
Since the `authenticationChallengeId` is being passed in the URL now we don't want to pass it in the request body.

Not sure how we'd do this in Java. Can we remove the `JsonProperty` attribute to exclude it, or do we need to specifically ignore it?